examples/advanced/quantum_volume.ipynb

```diff
@@ -0,0 +1,109 @@
+    "    # Making the plot look consistent.\n",
+    "    axs.set_ybound(0.4, 1)\n",
+    "    axs.set_xbound(1.5, 4.5)\n",
+    "    axs.xaxis.set_major_locator(plt.MultipleLocator(1))\n",
+    "    axs.set_xlabel(\"width/depth of model circuit m=d\")\n",
+    "    axs.set_ylabel(\"est. heavy output probability h(d)\")\n",
+    "    fig.suptitle('Experimental data for square quantum volume circuits')\n",
+    "    axs.legend(loc='lower left')\n",
+    "    \n",
+    "# Some set-up variables. Feel free to play with these.\n",
+    "num_repetitions = 11 # This is supposed to be >= 100.\n",
+    "depths = range(2, 5) # The depths and number of qubits\n",
+    "\n",
+    "# Keep track of the ideal and noisy probabilities.\n",
+    "average_ideals = []\n",
+    "average_noisies = []\n",
+    "\n",
+    "for depth in depths:\n",
+    "    num_qubits = depth # Square matrix.\n",
+    "    print(f\"Running simulation with {num_qubits} qubits and a depth of {depth}\")\n",
+    "    average_ideal = 0\n",
+    "    average_noisy = 0\n",
+    "    for i in range(num_repetitions):\n",
+    "        print(f\"    Repetition {i + 1} of {num_repetitions}\")\n",
+    "        # Generate a model circuit.\n",
+    "        model_circuit = quantum_volume.generate_model_circuit(\n",
+    "            num_qubits, depth, random_state=np.random.RandomState()\n",
+    "        )\n",
+    "        # Compute its heavy set.\n",
+    "        heavy_set = quantum_volume.compute_heavy_set(model_circuit)\n",
+    "        print(f\"        Heavy Set: {heavy_set}\")\n",
+    "        \n",
+    "        # Run an ideal simulator over the model circuit.\n",
+    "        ideal = quantum_volume.sample_heavy_set(model_circuit, heavy_set)\n",
+    "        average_ideal += ideal\n",
+    "        print(f\"        Ideal simulation HOG probability: {ideal}\")\n",
+    "\n",
+    "        # Run a noisy simulator over the model circuit.\n",
+    "        noisy = quantum_volume.sample_heavy_set(model_circuit, heavy_set, sampler=cirq.DensityMatrixSimulator(noise=cirq.ConstantQubitNoiseModel(qubit_noise_gate=cirq.DepolarizingChannel(p=0.005))))\n",
+    "        print(f\"        Noisy simulation HOG probability: {noisy}\")\n",
+    "        average_noisy += noisy\n",
+    "        \n",
+    "        # TODO: The next steps are to compile the model circuit and run it on various devices.\n",
+    "\n",
+    "    # Compute the average performance over the total number of runs.\n",
+    "    average_ideal /= num_repetitions\n",
+    "    average_noisy /= num_repetitions\n",
+    "    print(f\"    Average ideal HOG probability: {average_ideal}\")\n",
+    "    print(f\"    Average noisy HOG probability: {average_noisy}\")\n",
+    "    average_ideals.append(average_ideal)\n",
+    "    average_noisies.append(average_noisy)\n",
+    "\n",
+    "# Create the chart.\n",
+    "generate_chart(depths, average_ideals, average_noisies)"
```

Comment:
Yeah I'd put `generate_chart` in its own cell so that you can collapse the printed output from the actual run and then have the plot show up without having to scroll so far

Reply:
Done